### DISCUSSION TITLE
How to stop auto-sync in the better note sync manager interface?

### DISCUSSION THREAD

**Discussion by QuinnYates** [windingwind/zotero-better-notes#1206] [Q&A]
Hi Better Notes community, I cannot locate the option to stop auto-syncing notes. I'm using Zotero 7 on a MacBook with the latest better note 2.1.6. plugin. The unsync button has greyed out and I'm not sure if it's normal.
![Screenshot 2024-11-20 at 4 55 24 pm](https://github.com/user-attachments/assets/f7b301a4-c93a-4725-a9ce-2676e818a97a)

**Comment by windingwind** [comment]
You need to first select some notes in the list
